test/functional/discovered_hosts_controller_test.rb

```diff
@@ -275,7 +275,7 @@ def test_multiple_reboot_failure
     ::ForemanDiscovery::NodeAPI::PowerService.any_instance.expects(:reboot).returns(false)
     post :submit_multiple_reboot, params: {:host_ids => host.id}, session: set_session_user(User.current)
     assert_redirected_to discovered_hosts_url
-    assert_equal "Errors during reboot: #{host.name}: failed to reboot", flash[:error]
+    assert_equal "{:message=>\"Errors during reboot: #{host.name}: failed to reboot\", :host_details=>[{:name=>\"#{host.name}\", :error=>\"macaabbccddeeff: failed to reboot \"}]}", flash[:error]
     assert_nil flash[:success]
```

Comment:
Is it correct to display JSON via flash hash? Flash goes directly to page via notification box, so it won't render correctly. I think the controller should format this into plain string in some way.